### PR TITLE
addressing test dependencies with -DCATKIN_ENABLE_TESTING=0

### PR DESCRIPTION
**Pull request by bulwahn**

### CODE REVIEW

File: gripper_action_controller/CMakeLists.txt [hunk 1]
```diff
@@ -17,7 +17,6 @@ find_package(catkin
       realtime_tools
       control_msgs
       trajectory_msgs
-      rostest
```
Comment (adolfo-rt) [inline gripper_action_controller/CMakeLists.txt:20]: You might want to also remove the `<build_depends>` element from the `package.xml`.

**Comment by adolfo-rt** [comment]
See one inline comment, otherwise +1

File: gripper_action_controller/CMakeLists.txt (hunk 1 above)
Comment (bulwahn) [inline gripper_action_controller/CMakeLists.txt:20]: You are right, and I actually did remove the element from the package.xml in just that commit.

**Comment by bulwahn** [comment]
See inline comment: the issue mentioned was already addressed.

**Comment by adolfo-rt** [comment]
All good then. Merging.

**Pull request merged**